README.md

```diff
@@ -44,6 +44,19 @@ without having to build it from source.
 TODO(https://github.com/mc-imperial/dredd/issues/28): add instructions
 on how to use Dredd
 
+`pi.cc` is a small example in `examples/simple` that calculates an approximation of pi. To run this example, run the following from the root of the repository:
```

Comment:
I suggest starting with a preamble:

We first show how to apply Dredd to a simple stand-alone program. We then show how to apply it to a C++ CMake project.

Before trying these instructions out, make sure you have built Dredd and that the `dredd` executable is located under [the third_party clang bin dir]

### Applying Dredd to a single file example